beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -38,6 +38,28 @@ const (
 	broadcastValidationConsensusAndEquivocation = "consensus_and_equivocation"
 )
 
+// PublishBlindedBlock instructs the beacon node to use the components of the `SignedBlindedBeaconBlock` to construct
+// and publish a SignedBeaconBlock by swapping out the transactions_root for the corresponding full list of `transactions`.
+// The beacon node should broadcast a newly constructed SignedBeaconBlock to the beacon network, to be included in the
+// beacon chain. The beacon node is not required to validate the signed BeaconBlock, and a successful response (20X)
+// only indicates that the broadcast has been successful. The beacon node is expected to integrate the new block into
+// its state, and therefore validate the block internally, however blocks which fail the validation are still broadcast
+// but a different status code is returned (202). Pre-Bellatrix, this endpoint will accept a SignedBeaconBlock. After
+// Deneb, this additionally instructs the beacon node to broadcast all given signed blobs.
+func (s *Server) PublishBlindedBlock(w http.ResponseWriter, r *http.Request) {
+	ctx, span := trace.StartSpan(r.Context(), "beacon.PublishBlindedBlock")
+	defer span.End()
+	if shared.IsSyncing(r.Context(), w, s.SyncChecker, s.HeadFetcher, s.TimeFetcher, s.OptimisticModeFetcher) {
+		return
+	}
+	isSSZ, err := http2.SszRequested(r)
+	if isSSZ && err == nil {
```

Comment:
We are basically ignoring the error, which is a code smell. I think it would be better to change the code in `SszRequested` to not return the error at all:
```
priority, err := strconv.ParseFloat(match[0][1], 32)
if err != nil {
	return false
}
```